src/ninja.cc

```diff
@@ -892,7 +892,12 @@ std::string EvaluateCommandWithRspfile(const Edge* edge,
     rspfile_content.replace(newline_index, 1, 1, ' ');
     ++newline_index;
   }
-  command.replace(index - 1, rspfile.length() + 1, rspfile_content);
+  if (command[index - 1] == '@')
```

Comment:
Please add {}, see https://google.github.io/styleguide/cppguide.html#Formatting_Looping_Branching

Reply:
I used clang-format for this as well, hope that's fine now